plugin_runner/sandbox.py

```diff
@@ -126,6 +126,7 @@ class ImportedNames(TypedDict):
     "json": {
         "dumps",
         "loads",
+        "JSONDecodeError"
```

Comment:
we will want to add these to https://docs.canvasmedical.com/sdk/sandboxing-and-allowed-imports/